src/sentry/conf/server.py

```diff
@@ -1788,6 +1788,7 @@ def get_sentry_sdk_config():
     "sentry.web.api",
     "sentry.web.frontend",
     "sentry.api.endpoints",
+    "sentry.data_export.endpoints",
```

Comment:
added to the `SENTRY_REQUEST_METRIC_ALLOWED_PATHS` since it kinda looked right, not sure what exactly this does lol

Reply:
This is for automatically capturing performance stats in datadog. Ideally we'll change this in the future to be a blacklist, since we only care about not including ingest related things here.